appveyor.yml

```diff
@@ -50,8 +50,6 @@ build_script:
 
 test_script:
   - echo test_script
-  - set PROJ_LIB=%PROJ_DIR%\share\proj
-  - cd %PROJ_LIB%
```

Comment:
this cd is currently needed so that the grids are decompressed in the right directory